docs/user-guide/configure-certificates-keyring.md

```diff
@@ -25,7 +25,7 @@ To customize the `ZWEKRING` JCL, edit the JCL variables at the beginning of the
 
 ### `PRODUCT` variable
 
-The `PRODUCT` variable specifies the z/OS security manager.  The default value is `RACF`. Change the value to `ACF2` or `TSS` if you are using Access Control Facility CA-ACF2 or CA Top Secret for z/OS as your z/OS security manager.  
+The `PRODUCT` variable specifies the z/OS security manager.  The default value is `RACF`. Change the value to `ACF2` or `TSS` if you are using Access Control Facility CA-ACF2 or Top Secret for z/OS as your z/OS security manager.  
```

Comment:
Should this `CA-ACF2` be rebranded as well?

Reply:
Thanks a lot! Addressed with another commit :)